packages/pwa/app/ssr.js

```diff
@@ -30,7 +30,7 @@ const app = createApp({
 
     // This is the value of the 'mobify' object from package.json
     // provided by a webpack DefinePlugin
-    mobify: WEBPACK_PACKAGE_JSON_MOBIFY,
+    // mobify: WEBPACK_PACKAGE_JSON_MOBIFY,
```

Comment:
We're not getting rid of this `mobify` option to the SSR Server – at least, not in V1. We should continue to pass the config object here though, rather than doing `getConfig` inside the server-class itself. We maintain flexibility by injecting these dependencies.

If you don't like the name, we can fix _that_ in V2 pretty easily!

Reply:
Yeah.. I agree at least for v1 that we pass the loaded config through this option. I do however like the idea of having both the flexibility of being able to pass in this config but fall back to the loaded version for v2. For now I'll revert that change.